libexec/rc/rc.conf

```
@@ -47,6 +47,7 @@ devmatch_blacklist=""	# List of modules (w/o .ko) to exclude from devmatch.
 kldxref_enable="YES"	# Build linker.hints files with kldxref(8).
 kldxref_clobber="NO"	# Overwrite old linker.hints at boot.
 kldxref_module_path=""	# Override kern.module_path. A ';'-delimited list.
+modules="libiconv libmchain msdosfs_iconv if_bwn if_bwi runfw if_run if_rum if_iwm cuse geom_uzip fdescfs iwn4965fw iwn1000fw iwn5000fw iwn5150fw iwn6000fw iwn6000g2afw iwn6000g2bfw iwn6050fw if_urtw rtwn acpi_video fuse ums atp"
```

```suggestion
modules="cuse geom_uzip fdescfs fuse ums"
```
Remove all the network modules (auto-loaded by the "devmatch" service now), and also remove desktop-oriented modules (TrueOS is a building platform/server only now).